internal/cluster/cluster.go

```diff
@@ -788,10 +789,31 @@ func (m *Manager) DeleteClusterFiles(ctx context.Context, c *common.Cluster, obj
 	return nil
 }
 
-func (m Manager) PermanentClustersDeletion(ctx context.Context, olderThen strfmt.DateTime, objectHandler s3wrapper.API) error {
+func (m Manager) DeregisterInactiveCluster(ctx context.Context, maxDeregisterPerInterval int, inactiveSince strfmt.DateTime) error {
+	log := logutil.FromContext(ctx, m.log)
+
+	var clusters []*common.Cluster
+	db := m.db.Unscoped()
```

Comment:
You don't need this. 
It will get deleted clusters that are not relevant for this function.